### DISCUSSION TITLE
So You guys fired your 2nd highest contributor.what happened?

### DISCUSSION THREAD

**Discussion by liprais** [toeverything/AFFiNE#4572] [Q&A] [answered]
As titled.

**Comment by HeJiachen-PM** [comment, marked as answer]
Hi. I am not sure who exactly you are referring to. There are people who come and leave. We at AFFiNE have always been trying to keep a small team for better sustainability. 
We hope to focus on the product feedback and practices about AFFiNE with this Q&A section. No personal information will be discussed here as it is not of general interest to others. 
Should you have more questions about our company, toeverything .LTD, please send a email to our email contact@toeverything.info or my email address: hejiachen [at] toeverything.info.

Cheers,
Jiachen, founder of AFFiNE
